addons/account/models/account_move_line.py

```diff
@@ -1489,7 +1489,7 @@ def _valid_field_parameter(self, field, name):
     def _unlink_except_posted(self):
         # Prevent deleting lines on posted entries
         if not self._context.get('force_delete') and any(m.state == 'posted' for m in self.move_id):
-            raise UserError(_("You can not delete a posted journal item. Don’t play Jenga with your accounting records; reset the journal entry to draft before deleting."))
+            raise UserError(_("You cannot delete a posted journal item. Don’t play chess with your accounting records; reset the journal entry to draft before deleting."))
```

Comment:
as someone who can play chess, I have no idea what this mean in this context. Playing jenga made some sense, since jenga is a game where the pile can collapse at any moment (but it may not be well known in some countries?), however, I really don't see how this sentence make sense with the chess game.

Reply:
Hello, @ged-odoo  I wanted to make an addition as I couldn't come up with anything myself. In the next stage, we might receive some suggestions from the Product Owners, which is why I made this change for now.